src/app/components/elements/modals/EmailConfirmationModal.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  closeActiveModal,
+  requestEmailVerification,
+  selectors,
+  useAppDispatch,
+  useAppSelector,
+} from "../../../state";
+import { Button, CardBody } from "reactstrap";
+import React from "react";
+import { Link } from "react-router-dom";
+import { WEBMASTER_EMAIL } from "../../../services";
+
+const EmailConfirmationModalBody = () => {
+  const dispatch = useAppDispatch();
+  const user = useAppSelector(selectors.user.orNull);
+  const status = (user?.loggedIn && user?.emailVerificationStatus) || null;
+  
+  function clickVerify() {
+    dispatch(requestEmailVerification());
+    dispatch(closeActiveModal());
+  }
+
```

Comment:
surplus line feed. Might as well clean it up, the linter will complain later anyway